docs/Meadow/Release_Notes/v1/index.md

```diff
@@ -10,6 +10,30 @@ subtitle: Release Notes
 * [Meadow.CLI](/Meadow/Meadow_Tools/Meadow_CLI/)
 * [Meadow.OS](/Meadow/Getting_Started/Deploying_Meadow%2EOS/)
 
+## v1.15.0.0
+
+This release delivers some key optimizations to Meadow networking and sleep.
+
+### Meadow.OS
+
+* Fixed a critical issue that was affecting the reliability of Meadow Sleep/Wake
+* Added support for the Quectel EG21 modem
+* Minor improvements to edge-case network reliability
+
+### Meadow.OS

```

Comment:
Should this be a different header than the prior one? Or should they be combined?

Reply:
fixed!